assets/stylesheets/comments.scss

```diff
@@ -75,6 +85,34 @@
         padding-left: .25em;
     }
 
+    .openqa-flag {
+        position: relative;
+        background-color: #fdd596;
+        color: #666666;
+        padding: .3em;
+        padding-left: 1.25em;
+        border-radius: .25rem;
+        border: 1px solid #ecd785;
+    }
+
+    .openqa-flag::before {
+        display: inline-block;
+        font-style: normal;
+        font-variant: normal;
+        text-rendering: auto;
+        font-family: "ForkAwesome";
+        font-weight: 900;
+        font-size: .8em;
+        content: "\f024";
+        position: absolute;
+        left: 0;
+        top: 50%;
+        transform: translateY(-50%);
+        width: 1.25em;
+        text-align: center;
+        padding-left: .25em;
+    }
```

Comment:
This looks identical to `.openqa-label::before` except for `content` so I also suggest you make a common block for them instead of C&P.

Reply:
done